notebooks/Emukit-tutorial-basic-use-of-the-library.ipynb

```diff
@@ -66,9 +66,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 3,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "ImportError",
```

Comment:
That's not nice! Can you rerun this notebook to get rid of errors (there is one more below)